docs/DebuggingTheCompiler.md

```diff
@@ -807,6 +812,74 @@ reducing SIL test cases by:
 For more information and a high level example, see:
 ./swift/utils/bug_reducer/README.md.
 
+## Debugging the Compiler Driver
+
+The Swift compiler uses a standalone compiler-driver application:
+[swift-driver](https://github.com/apple/swift-driver) When building the compiler
+using `build-script`, by default, the standalone driver will be built first,
+using the host toolchain. Once the compiler is built, the compiler build
```

Comment:
I would add "if the host toolchain contains swift. Today, we will not error if the host toolchain does not contain swift, but that may change in the future.'

Reply:
Added clarification. Thanks.